axiom/nr_txn.c

```diff
@@ -3496,4 +3496,6 @@ void nr_txn_add_php_package(nrtxn_t* txn,
 
   p = nr_php_package_create(package_name, package_version);
   nr_php_packages_add_package(txn->php_packages, p);
+  nrl_verbosedebug(NRL_INSTRUMENT, "Detected package %s, version %s",
+                   package_name, package_version);
```

Comment:
The agent already logs this message [here](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/13b4a7e7bf8ef2f3bdcaab3f138d42cfb6e586f3). I think this is going to be too verbose.

Reply:
Right we added that one recently to the other PR - I can remove this one.